File: src/main/java/com/gra/paradise/botattendance/model/MissionType.java
package com.gra.paradise.botattendance.model;

import lombok.Getter;

@Getter
public enum MissionType {
    PATROL("Patrulhamento"),
    ACTION("Ação"),
    OTHER("Outros");

    private final String displayName;

    MissionType(String displayName) {
        this.displayName = displayName;
    }
}

File: src/domain/enums/MissionType.js
/**
 * Mission types available for operations
 * @readonly
 * @enum {string}
 */
const MissionType = Object.freeze({
  CAS: 'CAS',           // Close Air Support
  CAP: 'CAP',           // Combat Air Patrol
  SEAD: 'SEAD',         // Suppression of Enemy Air Defenses
  STRIKE: 'STRIKE',     // Strike Mission
  TRANSPORT: 'TRANSPORT', // Transport Mission
  TRAINING: 'TRAINING', // Training Mission
  ACTION: 'ACTION',     // Action Mission
  OUTROS: 'OUTROS'      // Other/Custom Mission
});

export default MissionType;